### PR TITLE
fix UWP build issues

### PR DESCRIPTION
**Pull request by Iguins**
## PR Type
What kind of change does this PR introduce?

Fix for
Error       Unexpected 'NONE' in parse rule 'Element ::= . EmptyElement | ( StartElement ElementBody ).'.   Microsoft.Toolkit.Uwp.SampleApp

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../readme.md#supported)
- [ ] Pull Request has been submitted to the documentation repository [instructions](..\contributing.md#docs). Link: 
- [ ] Sample in sample app has been added / updated (for bug fixes / features)
    - [ ] Icon has been created (if new sample) following the [Thumbnail Style Guide and templates](https://github.com/windows-toolkit/WindowsCommunityToolkit-design-assets)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Header has been added to all new source files (run *build/UpdateHeaders.bat*)
- [ ] Contains **NO** breaking changes

### CODE REVIEW

File: Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxPage.xaml.cs [hunk 1]
```diff
@@ -209,23 +209,23 @@ private void TokenItemAdded(TokenizingTextBox sender, object data)
             // TODO: Add InApp Notification?
             if (data is SampleDataType sample)
             {
-                Debug.WriteLine("Added Token: " + sample.Text);
+                //Debug.WriteLine("Added Token: " + sample.Text);
```
Comment (jeromelaban) [inline Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxPage.xaml.cs]:
```suggestion
                System.Diagnostics.Debug.WriteLine("Added Token: " + sample.Text);
```

File: Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj [hunk 2]
```diff
@@ -18,6 +18,10 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <PackageReference Include="Uno.UI" Version="3.0.17" />
+	</ItemGroup>
+
+	<ItemGroup>
```
Comment (jeromelaban) [inline Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj]: The Uno.UI package is defined above in all projects.

Comment (GuillaumeSE) [reply Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj:24]: Sure i'm going to update to a more recent one.

Comment (Iguins) [reply Microsoft.Toolkit.Uwp.Connectivity/Microsoft.Toolkit.Uwp.Connectivity.csproj]: Oups got the mistake 👍

File: Microsoft.Toolkit.Uwp.SampleApp/App.xaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-			 xmlns:not_wasm="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:not_wasm="http://uno.ui/not_wasm"
              mc:Ignorable="d not_wasm"
```
Comment (MatFillion) [inline Microsoft.Toolkit.Uwp.SampleApp/App.xaml]: not wasm should include windows, so placing it in ignorable is wrong.

Comment (MatFillion) [reply Microsoft.Toolkit.Uwp.SampleApp/App.xaml]: If you remove it from Ignorable, then you need to keep  `xmlns:not_wasm="http://schemas.microsoft.com/winfx/2006/xaml/presentation"`

File: Microsoft.Toolkit.Uwp.SampleApp/App.xaml [hunk 4]
```diff
@@ -16,7 +16,7 @@
 				<ResourceDictionary Source="ms-appx:///Styles/Generic.xaml" />
 
 				<!-- WinUI -->
-				<not_wasm:XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
```
Comment (MatFillion) [inline Microsoft.Toolkit.Uwp.SampleApp/App.xaml:19]: @davidjohnoliver or @jeromelaban  any idea why there's a not_wasm for the winui resources?

File: Microsoft.Toolkit.Uwp.SampleApp/App.xaml (hunk 3 above)
Comment (Iguins) [reply Microsoft.Toolkit.Uwp.SampleApp/App.xaml]: Alright fixed!

File: Microsoft.Toolkit.Uwp.SampleApp/App.xaml (hunk 4 above)
Comment (davidjohnoliver) [reply Microsoft.Toolkit.Uwp.SampleApp/App.xaml:19]: It's a recent PR https://github.com/unoplatform/Uno.WindowsCommunityToolkit/pull/105 - I gather it was package-size-related but @jeromelaban may have more details?

File: Microsoft.Toolkit.Uwp.SampleApp/App.xaml [hunk 5]
```diff
@@ -16,7 +16,7 @@
...
+				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
```
Comment (davidjohnoliver) [inline Microsoft.Toolkit.Uwp.SampleApp/App.xaml:19]:
```suggestion
				<not_wasm:ResourceDictionary>
					<ResourceDictionary.MergedDictionaries>
						<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
					</ResourceDictionary.MergedDictionaries>
				<not_wasm:ResourceDictionary/>
```

Comment (jeromelaban) [reply Microsoft.Toolkit.Uwp.SampleApp/App.xaml:19]: Indeed, this was size related for Wasm. It probably still is required.

**Pull request closed without merging**